packaging/installer/methods/kubernetes.md

```diff
@@ -8,105 +8,130 @@ learn_topic_type: "Tasks"
 learn_rel_path: "Installation"
 -->
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Deploy Kubernetes monitoring with Netdata
 
-This document details how to install Netdata on an existing Kubernetes (k8s) cluster. By following these directions, you
-will use Netdata's [Helm chart](https://github.com/netdata/helmchart) to create a Kubernetes monitoring deployment on
-your cluster.
+This document details how to install Netdata on an existing Kubernetes (k8s) cluster, and connect it to Netdata Cloud.  
```

Comment:
add to the end of this sentence add "See what you'll get" or something similar and remove the next sentence.